app/app.go

```diff
@@ -785,7 +789,10 @@ func initParamsKeeper(appCodec codec.BinaryMarshaler, legacyAmino *codec.LegacyA
 	return paramsKeeper
 }
 
-func (app *App) registerUpgradeHandlers() {
+// registerUpgradeHandlers registers upgrade handlers, and sets the storetypes loader if necessary.
```

Comment:
```suggestion
// registerUpgradeHandlers registers upgrade handlers, and sets the store loader if necessary.
```
Sorry. Could you fix this typo?

Reply:
Thank you. This is probably what I changed wrong.